keras/applications/efficientnet.py

```diff
@@ -852,10 +852,10 @@ def preprocess_input(x, data_format=None):
 
     Args:
       x: A floating point `numpy.array` or a `tf.Tensor`.
-      data_format: Optional data format of the image tensor/array. Defaults to
-        None, in which case the global setting
-        `tf.keras.backend.image_data_format()` is used (unless you changed it,
-        it defaults to "channels_last").{mode}
+      data_format: Optional data format of the image tensor/array. `None` means
+        the global setting `tf.keras.backend.image_data_format()` is used
+        (unless you changed it, it uses "channels_last").{mode}.
```

Comment:
Same here.

Reply:
@qlzh727 The issue is that `Defaults to` also needs to appear at the end of the string. Actually are you sure `{mode}` is only replaced when it appears at the end of the string? - Where does this happen? - Happy to modify that interpolation code.

Reply:
Well. I think they are a typo here then. We can remove the {mode}.

https://www.tensorflow.org/api_docs/python/tf/keras/applications/convnext/preprocess_input shows that the {mode} is not replaced properly.

It was replaced in some other application models, eg https://github.com/keras-team/keras/blob/901950201d867c85ec34f4d0c9201aea2c15a65d/keras/applications/imagenet_utils.py#L126